src/hooks/journals/useJournalNameForLogs.ts

```diff
@@ -1,21 +1,25 @@
-import { useEntityType } from 'context/EntityContext';
 import { useMemo } from 'react';
+import { DEFAULT_FILTER } from 'services/supabase';
+import { ShardEntityTypes } from 'stores/ShardDetail/types';
+import { hasLength } from 'utils/misc-utils';
 
 // This will expand eventually. Federated data planes will need to write logs to
 //      different regions. This is why this is in an array and takes some kind of "region selector".
 const REGIONS = [`us-central1`];
 
-function useJournalNameForLogs(entityName: string, region: number = 0) {
-    const entityType = useEntityType();
-
+function useJournalNameForLogs(
+    entityName: string,
+    entityType: ShardEntityTypes[],
+    region: number = 0
+) {
     return useMemo(() => {
-        console.log('useJournalNameForLogs');
         const collectionName = `ops.${REGIONS[region]}.v1/logs`;
-
         return [
-            `${collectionName}/kind=${entityType}/name=${encodeURIComponent(
-                entityName
-            )}/pivot=00`,
+            !hasLength(entityType)
+                ? ''
+                : `${collectionName}/kind=${
+                      entityType[0] ?? DEFAULT_FILTER
+                  }/name=${encodeURIComponent(entityName)}/pivot=00`,
```

Comment:
This is a bit overkill - but wanting to make it safe. Collections do not ever call this but wanted to make it safe.

Reply:
The defaulting to `DEFAULT_FILTER` is pretty overkill but wanted to make sure this was as safe as possible and make it easier to find calls that are wrong.